web/test/integration/homepage/projects.feature

```diff
@@ -5,27 +5,27 @@ Feature:  Display public projects on the homepage
     And there is a document "test-user-minimum" with the js "users/user-minimum" in collection "users"
     And there is a document "test-user-minimum" with the js "user-stats/user-minimum" in collection "userStats"
     When the "/" page is open
-    Then the text "Test public project with minimum data title" is in the element ".project__list__title"
-    And the text "0" is in the element ".mat-column-repository"
-    And the text "0" is in the element ".mat-column-monitors"
-    And the text "0" is in the element ".mat-column-pings"
-    And the text "test-user-minimum" is in the element ".mat-column-user"
-    And the text "1 second ago" is in the element ".mat-column-lastDate"
+    And find the row of text "Test public project with minimum data title"
+    Then the text "0" is in the row
```

Comment:
For this can we be even more specific and use the row as well as the cell?

Reply:
ok